document_page/static/src/js/document_page_kanban.js

```diff
@@ -0,0 +1,26 @@
+odoo.define("document_page.update_kanban", function (require) {
+    "use strict";
+
+    var KanbanRecord = require("web.KanbanRecord");
+
+    KanbanRecord.include({
+        // --------------------------------------------------------------------------
+        // Private
+        // --------------------------------------------------------------------------
+
+        /**
+         * @override
+         * @private
+         */
+        _openRecord: function () {
+            if (
+                this.modelName === "document.page" &&
+                this.$(".o_document_page_kanban_boxes a").length
+            ) {
+                this.$(".o_document_page_kanban_boxes a").first().click();
+            } else {
+                this._super.apply(this, arguments);
+            }
+        },
+    });
+});
```

Comment:
How about this? From odoo 16, 17 we have switched to using **`OWL framework`** for odoo js. Please switch it back.